### DISCUSSION TITLE
Separate subdomain logins

### DISCUSSION THREAD

**Discussion by electrofloat** [authelia/authelia#5145] [Q&A] [answered]
So.. I was trying to figure this out from the docs, and github issues (like this one: https://github.com/authelia/authelia/issues/1198) but I'm not sure it can be done, or just don't understand some config values.

What I would like to have is be able to login to `a.example.com` with user `X` and at the same time login to `b.example.com` with user `Y`.

And either in the above case, or if logged in to both subdomains with the same user, when I logout from a.example.com, only logout from that subdomain, and still be logged in on b.example.com.

Is this possible currently?

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for the late reply I'm just skimming things I've missed.

There is no direct support for this but similar to this a somewhat similar feature is coming: https://github.com/authelia/authelia/discussions/6119#discussioncomment-7355474
